assets/wizards/newspack/views/settings/connections/webhooks/endpoint-actions-card.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Settings Wizard: Connections > Webhooks > Endpoint Actions Card
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { getEndpointTitle } from './utils';
+import EndpointActions from './endpoint-actions';
+import WizardsActionCard from '../../../../../wizards-action-card';
+
+function EndpointActionsCard( {
+	endpoint,
+	setAction,
+}: {
+	endpoint: Endpoint;
+	setAction: ( action: WebhookActions, id: number | string ) => void;
+} ) {
+	return (
+		<WizardsActionCard
+			isMedium
+			className="newspack-webhooks__endpoint mt16"
+			toggleChecked={ ! endpoint.disabled }
+			toggleOnChange={ () => setAction( 'toggle', endpoint.id ) }
+			key={ endpoint.id }
+			title={ getEndpointTitle( endpoint ) }
+			disabled={ endpoint.system }
```

Comment:
I think this needs to be `endpoint.disabled`. Endpoints aren't able to be toggled off right now.

Reply:
Thanks! This is working now 👍